### DISCUSSION TITLE
onChange event for search module?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1870] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hello, I'm using search module to let users select one of the results. An event is triggered when they select, but how to get triggered every time the text they enter changes?
I need something like `onChange` event, that other modules have...

Any workaround?

**Comment by lubber-de** [comment, marked as answer]
Doesn't `onSearchQuery` callback work for you?
```javascript
$('.ui.search').search({
  onSearchQuery: function(val) {
    console.log('You typed ' + val);
  }
});
```
See https://jsfiddle.net/lubber/erhbwcxm/3/

**Reply by smartm0use** [reply]
Yes, I used it, thank you.
